### PR TITLE
PLAT-1245 - update common play template with detailed release slack n…

### PR DESCRIPTION
**Pull request by adrianvit**
- slack release notification
- fix log error metric
- clean up PR template

Should be merged/deploy along with https://github.com/telegraph/jenkins-pipeline-generator/pull/3

### CODE REVIEW

File: src/main/g8/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,31 +2,16 @@
 _[bug, improvement, feature]_
```
Comment (CristianDina) [inline src/main/g8/PULL_REQUEST_TEMPLATE.md]: I'd remove this line

File: src/main/g8/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,31 +2,16 @@
...
-https://jira.aws.telegraph.co.uk/browse/PLAT-880
+
 #### Outstanding questions?
-Might need tweaks to cpu/mem reservation.
-######Example:
-Might need tweaks to cpu/mem reservation.
+
 #### Definition of Done:
 - [ ] Is there appropriate test coverage?
 - [ ] Does the confluence need to be updated? https://jira.aws.telegraph.co.uk/browse/PLAT-926
```
Comment (CristianDina) [inline src/main/g8/PULL_REQUEST_TEMPLATE.md]: Also I'd remove links to other concrete tickets like PLAT-926 here in a template

Comment (adrianvit) [reply src/main/g8/PULL_REQUEST_TEMPLATE.md]: Fixed this, left the bug/improvement/feature

File: src/main/g8/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,31 +2,16 @@
...
 #### What's this PR do?
 _Sort description of the purpose of the changes in the pull request_
-######Example: 
-Add cloudformation templates for NAM ECS service. This will create two ECS services, one for warmer, one for worker, and a Redis replication group.
-Used the same Redis version and instance type as the current deployment.
-The way we startup the application has changed. Basically we will pass -Dmode=worker/warmer. This is documented in the readMe file.
-A docker file was also created, which takes a "MODE" build argument. This is used to update the new relic config and set the jvm properties for the spring boot app.
-Used official openjdk 8 alpine based image https://hub.docker.com/_/openjdk/ https://github.com/docker-library/openjdk/blob/4e39684901490c13eaef7892c44e39043d7c4bed/8-jre/alpine/Dockerfile
 #### Where should the reviewer start?
 _Guide for the reviewer or where to start_
```
Comment (CristianDina) [inline src/main/g8/PULL_REQUEST_TEMPLATE.md]: I'd also remove descriptions from each subtopic, since they mostly repeat the above sentence

**Pull request merged**